nvtabular/inference/triton/__init__.py

```diff
@@ -250,6 +285,183 @@ def _generate_model_config(
     return config
 
 
+def _generate_ensemble_config(name, output_path, nvt_config, nn_config):
+    config = model_config.ModelConfig(
+        name=name + "_ens", platform="ensemble", max_batch_size=nvt_config.max_batch_size
+    )
+    config.input.extend(nvt_config.input)
+    config.output.extend(nn_config.output)
+
+    nvt_step = model_config.ModelEnsembling.Step(model_name=nvt_config.name, model_version=-1)
+    for input_col in nvt_config.input:
+        nvt_step.input_map[input_col.name] = input_col.name
+    for output_col in nvt_config.output:
+        nvt_step.output_map[output_col.name] = output_col.name + "_nvt"
+
+    tf_step = model_config.ModelEnsembling.Step(model_name=nn_config.name, model_version=-1)
+    for input_col in nn_config.input:
+        tf_step.input_map[input_col.name] = input_col.name + "_nvt"
+    for output_col in nn_config.output:
+        tf_step.output_map[output_col.name] = output_col.name
+
+    config.ensemble_scheduling.step.append(nvt_step)
+    config.ensemble_scheduling.step.append(tf_step)
+
+    with open(os.path.join(output_path, "config.pbtxt"), "w") as o:
+        text_format.PrintMessage(config, o)
+    return config
+
+
+def _generate_hugectr_ensemble_config(name, output_path, nvt_config, hugectr_config):
+    config = model_config.ModelConfig(name=name, platform="ensemble")
+    config.input.extend(nvt_config.input)
+    config.output.extend(hugectr_config.output)
+
+    nvt_step = model_config.ModelEnsembling.Step(model_name=name + "_nvt", model_version=-1)
+    for input_col in nvt_config.input:
+        nvt_step.input_map[input_col.name] = input_col.name
+    for output_col in nvt_config.output:
+        nvt_step.output_map[output_col.name] = output_col.name + "_nvt"
+
+    hugectr_step = model_config.ModelEnsembling.Step(model_name=name + "hctr", model_version=-1)
+    for input_col in hugectr_config.input:
+        hugectr_step.input_map[input_col.name] = input_col.name + "_nvt"
+    for output_col in hugectr_config.output:
+        hugectr_step.output_map[output_col.name] = output_col.name
+
+    config.ensemble_scheduling.step.append(nvt_step)
+    config.ensemble_scheduling.step.append(hugectr_step)
+
+    with open(os.path.join(output_path, "config.pbtxt"), "w") as o:
+        text_format.PrintMessage(config, o)
+    return config
+
+
```

Comment:
This function doesn't seem to be used anywhere - can we delete ?

```suggestion
```

Reply:
Yes, we don't use it. Deleting it. I wonder why flake8 didn't catch this :)